epub33/rs/index.html

```diff
@@ -1230,20 +1050,32 @@ <h4>The <code>rendition:page-spread-*</code> Properties</h4>
 				</section>
 			</section>
 
-			<section id="sec-fxl-html-dimensions">
-				<h3>HTML Initial Containing Block Dimensions</h3>
+			<section id="sec-fxl-dimensions">
+				<h3>Initial Containing Block Dimensions</h3>
 
-				<p>In this version of this specification, only the width and height expressions as defined in <a
-						href="https://www.w3.org/TR/epub-33/#sec-fxl-icb-html">Expressing in HTML</a> [[!EPUB-33]] MUST
-					be recognized by <a>Reading Systems</a>.</p>
+				<dl>
+					<dt>XHTML</dt>
+					<dd>
+						<p>In this version of this specification, <a>Reading Systems</a> MUST recognize only the width
+							and height expressions as defined in <a
```

Comment:
Why this "In this version of this specification"? I have not seen this type of reference elsewhere...

Reply:
> Why this "In this version of this specification"?

I think that was added because there wasn't a firm definition of the viewport meta when we first introduced this. We were still citing the apple developer docs. Sort of a subtle warning that there might be additional requirements in the future.

I agree it's not really needed, as everything is subject to change in the future.

Reply:
I'm also going to tweak the wording so that it's the same as the SVG ICB text.